share/schema_versions/App-Netdisco-DB-75-76-PostgreSQL.sql

```diff
@@ -0,0 +1,12 @@
+BEGIN;
+
+ALTER TABLE device ADD COLUMN "pae_control" text;
```

Comment:
I think file `lib/App/Netdisco/DB/Result/Device.pm` needs to have the extra column added to spec?